.eslintrc.js

```diff
@@ -193,6 +196,8 @@ module.exports = {
       'error',
       {
         devDependencies: [
+          '.eslintrc.js',
+          'streaming/.eslintrc.js',
```

Comment:
I am unsure why we need to include files from `streaming/` in the root config, it seems weird to me.

I will see if I find another way